.circleci/config.yml

```diff
@@ -57,13 +57,11 @@ workflows:
         matrix:
           parameters:
             postgres_image:
-            - circleci/postgres:11
-            - circleci/postgres:12
-            - circleci/postgres:13
-            - cimg/postgres:14.9
-            - cimg/postgres:15.4
-            - cimg/postgres:16.0
-            - cimg/postgres:17.0
+            - cimg/postgres:13.22
+            - cimg/postgres:14.19
+            - cimg/postgres:15.14
+            - cimg/postgres:16.10
+            - cimg/postgres:17.6
```

Comment:
Are we dropping support for pg 11 / 12 or is this just to stop using the circleci images?

Reply:
11/12 are EoL and unsupported by upstream.

We can continue to keep some support in the exporter, but we don't need to waste a lot of CI time on them.

Reply:
Thanks for the clarification